backends/arm/test/test_tosa.py

```diff
@@ -82,19 +82,28 @@ def prepare_model_and_ref(test_model, profile=TosaProfile.MI):
 
     model.eval()
     if profile == TosaProfile.BI:
+        permute_memory_to_nhwc = model.permute_memory_to_nhwc
         # Quantize the model
         captured_model_graph_module = capture_pre_autograd_graph(
             model, copy.deepcopy(model.inputs[profile])
         )
         # Setup the quantizer
         quantizer = XNNPACKQuantizer()
-        operator_config = get_symmetric_quantization_config(is_per_channel=False)
+
+        # TODO: remove this hack, haven't figure out entirely but the int8 mnv2
+        # can only pass if we set the weight_qmin=0 here
+        weight_qmin = 0 if test_model == "mobilenetv2" else -127
```

Comment:
we can remove this line if we set `quantize(Quantize(calibrate=False))` like [here](https://github.com/pytorch/executorch/blob/42eeebc7899295d76a235eab4563efd38c1feb98/backends/xnnpack/test/models/mobilenet_v2.py#L55)